SECURITY.md

```diff
@@ -0,0 +1,98 @@
+# Security Policy
+
+## Supported Versions
+
+The Jaeger project provides community support only for last minor version: bug fixes are released eiter as part of the next minor version or as an on-demand patch version. Independent of which version is next, all patch versions are cumulative, meaning that they represent the state of our `master` branch at the moment of the release. For instance, if the latest version is 1.19.0, bug fixes are released either as part of 1.20.0 or 1.19.1.
+
+Security fixes are given priority and might be enough to cause a new version to be released.
+
+## Reporting a Vulnerability
+
+_The following is a copy of the [Report a security issue](https://www.jaegertracing.io/report-security-issue/) page from our website. The website's version has precedence in case of conflicts._
+
+If you find something suspicious and want to report it, we'd really appreciate!
+
+### Ways to report
+
+* Prefer to always encrypt your message, no matter the channel you choose to report the issue
```

Comment:
Should this be changed to remove "Prefer to" - so "Always encrypt...."

Reply:
While I think most people reporting security issues would be comfortable with encryption in general, requiring them to encrypt might make them give up on reporting. I still prefer an unencrypted report than no report.

Reply:
"Where possible you should encrypt your message, especially if sent on a public channel. If encrypting the message is an issue, then please use a private channel (e.g. emailing a maintainer)"?